.eslintrc.json

```diff
@@ -1,36 +1,71 @@
 {
```

Comment:
Was this updated just to be consistent with runner?. If so, the runner specific rules should be removed, there are config that only apply to runner such as specifics relating to webdriver and chai etc.

Reply:
Yeah, I just lifted, and didn't update, same with version, although I did in packages.json